### PR TITLE
Deleted two UX positions for HUD

### PR DESCRIPTION
**Pull request by amandaschonfeld**
Already have enough applicants and need to process everyone before the deadline.

### CODE REVIEW

File: pages/open-positions.md [hunk 1]
```diff
@@ -32,8 +32,7 @@ If you’re interested in these opportunities, please email a detailed resume to
 
 **Department of Housing and Urban Development (HUD) Roles**
 
-- [User Experience Designer, Information Architecture]({{ site.baseurl }}/raas/hud/user-experience-designer-information-architecture/)
-- [User Experience Designer, Research]({{ site.baseurl }}/raas/hud/user-experience-designer-research/)
+
```
Comment (fureigh) [inline pages/open-positions.md]: An extra linebreak snuck in. Please delete it if you can. Thanks!

**Comment by fureigh** [comment]
@amandaschonfeld I took a look and added one line-specific comment. Please also delete these positions' respective files in the `_positions` directory by navigating to them and clicking the trash icon, then adding each of those changes to this pull request. ccing @rooneywp in case of questions. Thanks!

**Comment by pixelwake** [comment]
@fureigh I deleted the files and committed the changes to this branch.

**Comment by fureigh** [comment]
@rooneywp Looks good to me. Thanks!

**Comment by fureigh** [comment]
Made that remaining change and removed the priority consideration deadline, as it's passed. Merging.

**Pull request merged**